### ISSUE TITLE
SCSS Lint Integration

### ISSUE THREAD

**Issue by geebru** [bdkjones/CodeKit2#432]
Any chance we might see other Linting operations in the future, such as SCSS lint?

**Comment by bdkjones** [comment]
Well, it's really not needed because if you make any errors in your Sass file, it won't compile.

This is different than JavaScript, which is not a compiled language and therefore you can easily have errors make it to production code.

**Comment by geebru** [comment]
For outright errors yeah, but I mean in the realm of setting SCSS "rules" such as only use SCSS comments, lower case class names, etc. that are more personalized. That would require a significant addition though as options for each available checking point would be needed.

IE, more in the realm of checking for nice SCSS / standards rather than only errors.

**Comment by bdkjones** [comment]
Ah, I can see the use-case for style enforcement. Is there a good Sass linter available anywhere?

**Comment by geebru** [comment]
I've had my eye on this - https://github.com/causes/scss-lint

Still doing some tests with it when I have time in between projects but seems to be the best out there.

**Comment by Fequois** [comment]
@geebru that's an excellent implementation.

I would be interested in this feature as well, however, it is quite opinionated and will throw errors on you continuously. But I suppose those of us who are interested in this feature are also quite capable of setting it up as we want it.

**Comment by eriku** [comment]
I agree,  it would be a great feature to have (turned off by default). Especially with the capability of codekit2 having both a defaults and a per project config. I run scss-lint via the command line but would love to see the feature added to CodeKit so everything is in one place.

**Comment by fanfarian** [comment]
I'd love to see such a feature in Codekit.

**Comment by hpohlmeyer** [comment]
+1

**Comment by philval** [comment]
+1

**Comment by MysticEarth** [comment]
+1

**Comment by ilbonzo** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by nwpappas** [comment]
+1

Would be great for environments were multiple authors are hitting the SCSS on a regular basis.

**Comment by mlkstuff** [comment]
+1

**Comment by formspiel** [comment]
Any chance to see it in CK3? ;-)
